packages/frontend/src/components/UserCompletionModal/index.tsx

```diff
@@ -15,20 +15,29 @@ import {
     UserCompletionModalCard,
 } from './UserCompletionModal.styles';
 
-const jobTitles = [
+function shuffleArray(arr: string[]) {
+    return arr
+        .map((value) => ({ value, sort: Math.random() }))
+        .sort((a, b) => a.sort - b.sort)
+        .map(({ value }) => value);
+}
+
+const jobTitles: Array<string | OptionProps> = [
```

Comment:
you can remove the `Array<string | OptionProps>`  typing here, like before, this way we can remove the OptionProps import